resources/views/livewire/category-edit-form.blade.php

```diff
@@ -32,6 +28,18 @@
                     />
                     <span>{!! trans('admin/categories/general.use_default_eula') !!}</span>
                 </label>
+            @elseif ($eulaText!='')
+                <label class="form-control form-control--disabled">
+
+                    <input
+                            type="checkbox"
+                            name="use_default_eula"
+                            value="0"
+                            wire:model.live="shouldUncheckDefaultEulaBox"
```

Comment:
`wire:model` should be used for properties and not event listeners/handlers.

I _think_ ensuring `$useDefaultEula` is accurate is the correct fix?

Maybe we can make sure we're in a valid state from the start from within the `mount` method?

Reply:
I see this. For some reason i probably didn't do that combo of actions, cause I don't remember that happening.

Additionally, the checkmark for "use default EULA" persists on the category index page, even when no default EULA is made. so that is another spot we'd have to fix as well.

Reply:
or do we want to leave it there, but make sure the checkbox is always uncheckable if its checked, but then we don't allow it to be rechecked?

Reply:
so, yea, that checkmark WILL change upon another save of the edit page. so its not a huge issue, but could be confusing.

if there is no default eula set, then `useDefaultEula` should always be false. At no point it should ever turn true, I don't think it ever does.

So, it might just be a mess with reading the correct true/false and then presenting it on the page.